airbyte-config/init/src/main/java/io/airbyte/config/init/ApplyDefinitionsHelper.java

```diff
@@ -49,9 +48,9 @@ public void apply(final boolean updateAll) throws JsonValidationException, IOExc
     } else {
       // todo (pedroslopez): Logic to apply definitions should be moved outside of the
```

Comment:
@pedroslopez i think this PR does this todo. i don't have context into why it wasn't all just using `loadData` in the first place. is there any reason not to just reduce down to this one code path?

Reply:
The reason for this `updateAll` branching is because there are still some differences in how Cloud applies definitions vs OSS, but maybe there's a way to override the `updateActorDefinitions` method for Cloud instead of using this helper class. I think ideally we eventually standardize the behaviors between OSS and Cloud.